train-sim/kotlin_bindings/uniffi/train_sim/train_sim.kt

```diff
@@ -0,0 +1,997 @@
+// This file was autogenerated by some hot garbage in the `uniffi` crate.
+// Trust me, you don't want to mess with it!
```

Comment:
Cute :)

Do we really want to commit the bindings?

Reply:
Why not ? They're going to be used thoroughly in the end, so I figured we might as well make them visible, rather than having to dig in dynamically generated files ?